operator/deploy/operator.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pmem-csi
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pmem-csi-operator
+  namespace: pmem-csi
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: pmem-csi-operator
+  namespace: pmem-csi
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - services
+  - services/finalizers
+  - serviceaccounts
+  - endpoints
+  - events
+  - secrets
+  verbs:
+  - '*'
+- apiGroups:
+  - apps
+  resources:
+  - daemonsets
+  - statefulsets
+  verbs:
+  - '*'
+- apiGroups:
+  - apps
+  resourceNames:
+  - pmem-csi-operator
+  resources:
+  - deployments/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - roles
+  - rolebindings
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
```

Comment:
Do we need this `monitoring.coreos.com`?

Reply:
May be not, will remove.

Reply:
Removed this.